src/BuiltInTools/BrowserRefresh/BlazorHotReload.js

```diff
@@ -8,10 +8,12 @@ export function receiveHotReload() {
 export async function receiveHotReloadAsync() {
   const response = await fetch('/_framework/blazor-hotreload');
```

Comment:
Can you think of any way in which the implementor of _framework/blazor-hotreload is from a different SDK version but this injected JS code?

Reply:
I don't think that'd be a problem. We can remove this once we migrate ASP.NET to the newer API and insert it into the SDK. This is temporary so that we can make the changes incrementally.